src/Language/TokenReader.php

```diff
@@ -0,0 +1,777 @@
+<?php
+
+namespace Digia\GraphQL\Language;
+
+use Digia\GraphQL\Error\SyntaxErrorException;
+
+class TokenReader implements TokenReaderInterface
+{
+
+    /**
+     * The lexer owning this token reader.
+     *
+     * @var LexerInterface
+     */
+    protected $lexer;
+
+    /**
+     * {@inheritdoc}
```

Comment:
We have used `@inheritdoc`, but I guess this is also as correct. However, we should be consistent about this.